src/app/pages/AccountPage/index.tsx

```diff
@@ -57,16 +58,19 @@ interface NavItemProps {
   route: string
 }
 
+const counterZIndex = mobileHeaderZIndex - 1
 const NavItem = ({ counter, label, route }: NavItemProps) => {
   return (
     <StyledNavItem exact to={route}>
       <Text>{label}</Text>
       {!!counter && (
         <Box
-          style={{ position: 'absolute', top: '-3px', right: '-5px' }}
+          align="center"
+          style={{ position: 'absolute', top: '-3px', right: '-5px', zIndex: counterZIndex }}
```

Comment:
Commented on https://github.com/oasisprotocol/oasis-wallet-web/pull/937#discussion_r929082642
I can't reproduce an issue that would need z-index

Reply:
I will keep current version as I want to keep an overlap and in this case there is no benefit of adding extra div/Stack component where I still need a customised `StyledNavItem`